README.md

```diff
@@ -19,22 +20,27 @@ A Django application that lets users create forms. This is achieved firstly sett
     - [Contributing to the code](#contributing-to-the-code)
 
 ## What's in the box?
+
 The package contains the following:
-* Models to store forms, form questions and responses.
-* Views to create, edit, delete and submit forms.
-* Templates to render each of the views (can be overwritten with your own).
-* Ability to manage forms via the admin panel.
-* Endpoints to download questions and responses both directly via views and via the admin panel.
+
+- Models to store forms, form questions and responses.
+- Views to create, edit, delete and submit forms.
+- Templates to render each of the views (can be overwritten with your own).
+- Ability to manage forms via the admin panel.
+- Endpoints to download questions and responses both directly via views and via the admin panel.
 
 ## Installation
 
 ### PIP install and `settings.py`
+
 The application can be installed by running the following command:
-```
+
+```python
```

Comment:
It'd change this to "```bash" as strictly speaking it's not a python snippet as it's something we'd run in shell.

Reply:
Idk how I missed that, resolved on 7b9462fdabdf38e926bb4963649210fa873bd570